src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -361,6 +361,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 
 	private addDecoration(range: NotebookRange): void {
 		if (range && this.output && this.output.nativeElement) {
+			let markDoc = new Mark(this.output.nativeElement);
```

Comment:
I think the name could be better here - `markDoc` isn't very clear. Maybe `markOutput` or `outputMark`?